### ISSUE TITLE
ScheduledExecutorServiceTest.stats_manyRepetitionsTask

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#9574]
```
java.lang.AssertionError: expected:<6> but was:<5>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceTest.stats_manyRepetitionsTask(ScheduledExecutorServiceTest.java:209)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast/1143/testReport/junit/com.hazelcast.scheduledexecutor/ScheduledExecutorServiceTest/stats_manyRepetitionsTask/

**Comment by tkountis** [comment]
Suggested fix #9580

**Comment by tkountis** [comment]
#9580 merged.
